source/openapi-admin-v3.yaml

```diff
@@ -4387,6 +4387,137 @@ paths:
           description: No Content
 
+  "/groups/{groupId}/apps/{appId}/edge":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+
+    get:
+      tags:
+        - edge-server
+      operationId: adminGetAllEdgeServerInstances
+      summary: List Edge Servers
+      description: |-
+        List all [Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/) 
+        instances for the given App.
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  edge_servers:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/EdgeServerInstance"
+
+    post:
+      tags:
+        - edge-server
+      operationId: adminCreateEdgeServerInstance
+      summary: Create an Edge Server instance
+      description: Create a new Edge Server instance.
+      requestBody:
+        required: true
+        content:
+            application/json:
+              schema:
+                type: object
+                required: ["name"]
+                properties:
+                  name:
+                    type: string
+                    example: Edge-Store-42
+                    description: A human-readable name for the Edge Server instance.
+                    required: true
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EdgeServerCreateResponse"
+
+  "/groups/{groupId}/apps/{appId}/edge/{edgeServerId}":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/EdgeServerId"
+
+    get:
+      tags:
+        - edge-server
+      operationId: adminGetEdgeServerInstance
+      summary: Get Edge Server information
+      description: |-
+        Get information about a specific Edge Server instance.
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EdgeServerInfo"
+
+    delete:
+      tags:
+        - edge-server
+      operationId: adminDeleteEdgeServerInstance
+      summary: Delete an Edge Server instance
+      responses:
+        "200":
+          description: No content
+
+  "/groups/{groupId}/apps/{appId}/edge/{edgeServerId}/regenerate-registration-key": 
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/EdgeServerId"
+
+    post:
+      tags:
+        - edge-server
+      operationId: adminRegenerateEdgeServerRegistrationKey
+      summary: Regenerate Edge Server registration key
```

Comment:
we should update all instances of "registration key" with "registration token". except, unfortunately, the endpoint itself which we will keep as-is for now